README.md

```diff
@@ -730,6 +744,11 @@ An order for placing the rule in `pg_hba.conf`. Defaults to `150`.
 ####`target`
 This provides the target for the rule, and is generally an internal only property. Use with caution.
 
+####`postgresql_version`
+Defaults to the version set in `postgresql::server`.  Use this if you want to manage `pg_hba.conf` without managing the entire postgreSQL instance.
+
+####`manage_pg_hba_conf`
```

Comment:
@DavidS The purpose of the parameter is to allow setting it without needing to include `postgresql::server`.  My use case is to manage pg_hba_conf without managing the entire postgresql instance.  

I don't think this is the best long term solution but I'm hoping it's good enough for now.